components/menu/test/menu.test.js

```diff
@@ -115,6 +115,20 @@ describe('d2l-menu', () => {
 			expect(document.activeElement).to.equal(elem.querySelector('#b1'));
 		});
 
+		it('focus method sets focus to selected radio menu item', async() => {
```

Comment:
Nit: typically with the describe/it Mocha syntax, we try to make the `it` tests read like a sentence. So a different way to write the test name would be: `it('focuses on the selected radio menu item when focus() is called', ...`

Reply:
Sounds more useful :) though some other tests in that file aren't following that pattern.